packages/react-sdk/docusaurus/docs/React/01-basics/07-ui-customization-options.mdx

```diff
@@ -14,22 +14,22 @@ Here is a list of things still to address on this page:
 
 ## Introduction
 
-There are different ways to use the React Video SDK to build out UI. It comes with powerful built-in video components ready for use.
+There are different ways to use the React Video SDK to build out UI. It comes with powerful built-in components ready for use.
 In addition, it allows for styling and custom theming options with custom CSS.
 
-For custom use cases it is also possible to only use the low-level client to provide completely custom UI to fully tailor the SDK towards your needs.
+For custom use cases it is also possible to create your own components to fully tailor the SDK towards your needs. Please note that we suggest not to override a few, core components as those contain complex, low-level logic related to performance, the list of these components can be found in the [UI Components overview](../03-ui/01-overview.mdx).
 
 This page guides you through all of them with increasing levels of customization.
 
-### Using pre-built components
+## Using pre-built components
 
 The SDK comes with a lot of pre-built components for the different elements of the UI.
```

Comment:
But their customizability will be limited as we do not plan to allow passing Components to props.

Reply:
I am not sure how they will be able to customize the pre-built components.

Reply:
Some of them offer some props to tweak them. In any case there are options for styling them via CSS properties, right?

Reply:
I would conside the CSS part on the following section. I am still not sure how customizable they will be. Our intention is not to follow the stream-chat-react pattern, where you pass down components in props.

Reply:
Ok, let's revisit this later